.circleci/config.yml

```diff
@@ -59,11 +52,7 @@ workflows:
       - codacy/sbt:
           name: publish
           context: CodacyAWS
-          install_sbt_version: 1.3.3
           cmd: |
-            sbt ";retrieveGPGKeys"
-            sbt ";++2.11.12;clean;publishSigned;sonatypeRelease"
-            sbt -DplayVersion="2.7.4" ";++2.11.12;clean;publishSigned;sonatypeRelease"
-            sbt ";++2.12.10;clean;publishSigned;sonatypeRelease"
```

Comment:
I don’t think you can remove this. It is used in website and portal

Reply:
the missing part is the switch over `playVersion` if I read it correctly

Reply:
Right! Thought all versions were covered. Fixed :white

Reply:
We were publishing 3 before now you are only publishing 2. Are you sure the other is not needed?

Reply:
we are publishing 4 actually:
https://github.com/codacy/stash-scala-client/blob/master/build.sbt#L10

Reply:
with `+publishSigned` we are publishing the default play versions for the two Scala versions:
2.5 for Scala 2.11 and 2.7 for Scala 2.12
https://github.com/codacy/stash-scala-client/blob/b71f9bd8978ea73e847e4098c3abdee5039d0d79/project/Dependencies.scala#L9
Now I added one line to publish even for Play2.7+Scala2.11

Reply:
@lolgab that was what I was missing. That sneaky `+`

Reply:
should be good now

Reply:
> @lolgab that was what I was missing. That sneaky `+`

Yeah, these little operators are very easy to loose in code reviews!